content/en/docs/tasks/traffic-management/tcp-traffic-shifting/gtwapi_test.sh

```diff
@@ -18,7 +18,7 @@
 source "tests/util/gateway-api.sh"
 install_gateway_api_crds
 
-# @setup profile=none
+# @setup profile=demo
```

Comment:
This is a pretty kludgy fix for the failing tcp-traffic-shifting test. It's hiding a bigger problem. My guess is that there is some other test in the none profile that is messing something up the in the test cluster, which causes problems with tcp-traffic-shifting. Either that, or somehow the test really needs something that's set up in the demo install, but that's unlikely, given the test works fine running manually. We can rule out the second possibility by moving it to a different profile, 'minimal` for example, which would be closer to the required install anyway.
```suggestion
# @setup profile=minimal
```
If that works, then I think we can conclude that the problem is caused by some other test in the `none` profile.